src/assets/TokenListController.ts

```diff
@@ -54,7 +53,7 @@ type TokenListMessenger = RestrictedControllerMessenger<
   GetTokenListState,
   TokenListStateChange,
   never,
-  never
+  string
```

Comment:
This seems wrong. This generic parameter is for 'allowed events', it's meant to be set to specific events.

```suggestion
  never
```

Reply:
Updated to use the correct type